docs/usecases_content.rst

```diff
@@ -0,0 +1,52 @@
+
+Use-cases and Examples
+======================
+
+In this part of the user-manual, the most representative use-cases demonstrating :code:`integration-service`
+functionalities are shown.
+For each use-case, a related example is presented and the user is guided step-by-step through the
+installation protocol and environment preparation necessary to have the examples set up and working.
+
+Namely, we will go through the following:
+
++----------------------------------------------------------------------------------------------------+------------------------------+
+| Use-cases                                                                                          | Examples                     |
++====================================================================================================+==============================+
+| :ref:`DDS bridge <dds bridge>`                                                                     | :code:`ROS2`                 |
++----------------------------------------------------------------------------------------------------+------------------------------+
+| :ref:`Integrate a large system <integrate a large system>`                                         | :code:`Orion Context-Broker` |
++----------------------------------------------------------------------------------------------------+------------------------------+
+| :ref:`Add compatibility to an unsupported protocol <add compatibility to an unsupported protocol>` | :code:`SOME/IP`              |
++----------------------------------------------------------------------------------------------------+------------------------------+
+| :ref:`WAN communication <wan communication>`                                                       | WAN TCP tunneling            |
++----------------------------------------------------------------------------------------------------+------------------------------+
```

Comment:
It would be nice if the examples were a link to the example section instead of static text.

Reply:
done.